modules/porous_flow/test/tests/fluids/simple_fluid_yr_MPa_C_action.i

```diff
@@ -0,0 +1,114 @@
+# Version of simple_fluid_yr_MPa_C.i but using a PorousFlowFullySaturated Action, to check that the Action passes the unit choices through to the remainder of PorousFlow
+
+[Modules]
+  [./FluidProperties]
+    [./the_simple_fluid]
+      type = SimpleFluidProperties
+      thermal_expansion = 2.0E-4
+      cv = 4000.0
+      cp = 5000.0
+      bulk_modulus = 1.0E9
+      thermal_conductivity = 1.0
+      viscosity = 1.1E-3
+      density0 = 1500.0
+    [../]
+  [../]
+[]
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  nx = 1
+[]
+
+[GlobalParams]
+  PorousFlowDictator = dictator
+  gravity = '0 0 0'
+[]
+
+[Variables]
+  [./pp]
+    initial_condition = 10
+  [../]
+  [./T]
+    initial_condition = 26.85
+  [../]
+[]
+
+[PorousFlowFullySaturated]
+  coupling_type = ThermoHydro
+  porepressure = pp
+  temperature = T
+  temperature_unit = Celsius
+  pressure_unit = MPa
+  time_unit = years
+  fp = the_simple_fluid
+[]
+
+[Materials]
+  # these are needed by the Kernels, but are irrelevant to this particular problem
+  [./porosity]
+    type = PorousFlowPorosityConst
+    porosity = 0.2
+  [../]
+  [./zero_thermal_conductivity]
+    type = PorousFlowThermalConductivityIdeal
+    dry_thermal_conductivity = '0 0 0  0 0 0  0 0 0'
+  [../]
+  [./rock_heat]
+    type = PorousFlowMatrixInternalEnergy
+    specific_heat_capacity = 1.0
+    density = 1.0
+  [../]
+  [./permeability]
+    type = PorousFlowPermeabilityConst
+    permeability = '0 0 0 0 0 0 0 0 0'
+  [../]
+[]
+
+[Postprocessors]
+  [./pressure]
+    type = ElementIntegralVariablePostprocessor
+    variable = pp
+  [../]
+  [./temperature]
+    type = ElementIntegralVariablePostprocessor
+    variable = T
+  [../]
+  [./density]
+    type = ElementIntegralMaterialProperty
+    mat_prop = 'PorousFlow_fluid_phase_density_qp0'
```

Comment:
Maybe we should make a porous flow version of this in the future where the users don't have to know the internal names of our material properties. Not for this PR though.

Reply:
I don't think this is necessary, because most users will use `PorousFlowPropertyAux` to populate an AuxVariable, and then use standard MOOSE postprocessors.  This file uses this obscure stuff just because i (or you - can't remember who did this input file originally) wanted to short-circuit that process to keep the input file small.

Reply:
Good point. We probably did this first version before we had the aux kernels